utils/python/wait_for_tests.py

```diff
@@ -429,6 +429,12 @@ def wait_for_tests_impl(test_paths, no_wait=False, check_throughput=False, check
 
     return test_results
 
+def update_testspec_xml(test_results):
```

Comment:
I'm confused by this on many levels. Wait_for_tests already prints statuses. Also, why is this function called "update_testspec_xml" when it's just printing testnames?